### DISCUSSION TITLE
Generate the DDLs with `if not exists` or `if exists`

### DISCUSSION THREAD

**Discussion by shaounak** [sqlalchemy/alembic#1432] [Ideas]
Modern data warehouses supports `create table if not exists` or `drop table if exists`. Is there a way in alembic to run the migrations using this feature. e.g. in the op.create_table, can we create flag. If the value is set for the flag, then issue `CREATE TABLE IF NOT EXISTS`  else issue the default `CREATE TABLE` statement.

**Comment by CaselIT** [comment]
hi,

moving to issue to better track it

**Reply by CaselIT** [reply]
see https://github.com/sqlalchemy/alembic/issues/1434
